MODULE.bazel

```diff
@@ -0,0 +1,28 @@
+module(
+    name = "tools_android",
+    version = "0.2.0",
```

Comment:
i made a 0.2 release for the latest commit to support workspace, so let's make this 0.3

Reply:
@ahumesky thank you, done!

Reply:
I cleaned up a bit of the documentation as well to reflect that only bzlmod is supported, and that 0.2.0 is the last released version supporting WORKSPACE.